paper/paper.bib

```diff
@@ -16,34 +8,29 @@ @article{andresen_validation_2015
   pages = {1074--1088},
   issn = {03605442},
   doi = {10.1016/j.energy.2015.09.071},
-  file = {/home/fabian/Zotero/storage/DLUDQ586/Andresen et al. - 2015 - Validation of Danish wind time series from a new g.pdf;/home/fabian/Zotero/storage/FY9CBCGZ/Andresen et al. - 2015 - Validation of Danish wind time series from a new g.pdf;/home/fabian/Zotero/storage/MMDT3UAA/andresen2015.pdf;/home/fabian/Zotero/storage/QYAPCCB8/Andresen et al. - 2015 - Validation of Danish wind time series from a new g.pdf;/home/fabian/Zotero/storage/RZ26IHMM/andresen2015.pdf},
   journal = {Energy},
-  language = {en}
 }
 
 @article{beyer_robust_2004,
   title = {A Robust Model for the {{MPP}} Performance of Different Types of {{PV}}-Modules Applied for the Performance Check of Grid Connected Systems},
   author = {Beyer, Hans Georg and Heilscher, Gerd and Bofinger, Stefan},
   year = {2004},
-  file = {/home/fabian/Zotero/storage/BGKNK4EF/Beyer et al. - 2004 - A robust model for the MPP performance of differen.pdf;/home/fabian/Zotero/storage/BHXH68WX/Beyer et al. - 2004 - A robust model for the MPP performance of differen.pdf},
-  journal = {Eurosun. Freiburg}
+  journal = {EuroSun. Freiburg}
 }
 
 @article{brown_synergies_2018,
-  title = {Synergies of Sector Coupling and Transmission Extension in a Cost-Optimised, Highly Renewable {{European}} Energy System},
```

Comment:
Why the lowercase here? convention for titles is different right?

Reply:
Just happened to copy the entry from my thesis which had not capitalised the titles. Reverted now.